### PR TITLE
chore(component-testing): Adding typescript support, cy:run, and cy:open to runner-ct

### PR DESCRIPTION
**Pull request by JessicaSachs**
Previously we didn't have `cy:run` and `cy:open` commands in `runner-ct`. To run the component tests, we needed to precompile `webpack.config.ts` with tsc. This was annoying because it left *.js artifacts that would be accidentally required in subsequent runs.

I also removed the videos folder from showing up by default. We could .gitignore it instead. I'm fine either way.

How to test:
1. `yarn install`
2. `cd packages/runner-ct`
3. `yarn cypress:open # yarn cypress:run`

### CODE REVIEW

File: packages/runner-ct/cypress/plugins/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const path = require('path')
 module.exports = (on, config) => {
   on('dev-server:start', (options) => {
     // yarn tsc webpack.config.ts --esModuleInterop
```
Comment (lmiller1990) [inline packages/runner-ct/cypress/plugins/index.js:10]: I suppose we can delete this comment

File: packages/runner-ct/package.json [hunk 2]
```diff
@@ -8,6 +8,8 @@
     "build": "webpack",
     "build-prod": "cross-env NODE_ENV=production yarn build && tsc",
     "clean-deps": "rm -rf node_modules",
+    "cypress:open": "ts-node ../../scripts/start.js --component-testing --project ${PWD}",
```
Comment (lmiller1990) [inline packages/runner-ct/package.json:11]: I wonder how strict we want to be? We could add an additional `--transpile-only` flag to ignore and TS errors. But I am fine with this.

**Review by lmiller1990 — DISMISSED** [review]
two minor/options comments

**Comment by jennifer-shehane** [comment]
@lmiller1990 @JessicaSachs can we close this PR? Would like to stop being notified about this in GH PR notifications if it’s no longer relevant. Thanks 🙏

**Comment by lmiller1990** [comment]
@jennifer-shehane sure, I don't think we need this anymore, we already have TS support via some other PR.

If we do need this after all, we will make a new PR.

**Pull request closed without merging**